src/projects/nhs-collaboration.njk

```diff
@@ -5,6 +5,7 @@ templateEngineOverride: njk
 ogTitle: i.AI and NHS England Collaboration Charter
 ogImage: https://ai.gov.uk/img/nhs2.png
 ogUrl: https://ai.gov.uk/projects/nhs-collaboration
+ogDescription: "i.AI and NHS England sign Collaboration Charter to support the use of AI in the NHS"
```

Comment:
The description isn't used in the banner in this file

Reply:
Thanks Ell. I thought you might pick that one up!
It's a quick workaround so "NHS" doesn't appear on a line on it's own. I will tidy that up at some future point.